app/assets/javascripts/components/overview/yes_no_selector.jsx

```diff
@@ -11,10 +13,15 @@ const YesNoSelector = createReactClass({
     editable: PropTypes.bool,
     updateCourse: PropTypes.func.isRequired
   },
-
-  _handleChange(e) {
+  componentWillMount() {
```

Comment:
This can probably be done in `componentDidMount` rather than `componentWillMount`, which is not the recommend way to do it any longer: https://reactjs.org/docs/react-component.html#mounting

Reply:
Can we do it by getInitialState function ? https://reactjs.org/docs/react-without-es6.html

Reply:
Yes, getInitialState should work fine.